docs/source/reference/api-server/api-server-admin-deploy.rst

```diff
@@ -71,7 +71,7 @@ Following tabs describe how to configure credentials for different clouds on the
 
         .. code-block:: bash
 
-            helm upgrade --install skypilot skypilot/skypilot-nightly --devel \
+            helm -n $NAMESPACE upgrade --install skypilot skypilot/skypilot-nightly --devel \
               --set kubernetesCredentials.useKubeconfig=true \
```

Comment:
For consistency with the rest of the docs, should we use the `--namespace` flag instead and put it in a new line?
```suggestion
            helm upgrade --install skypilot skypilot/skypilot-nightly --devel \
              --namespace $NAMESPACE \
              --set kubernetesCredentials.useKubeconfig=true \
```

Reply:
Sounds good!